[2025-04-01 21:30:17,190] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: simple_test_dag.print_hello 2024-01-01T00:00:00+00:00 [queued]>
[2025-04-01 21:30:17,210] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: simple_test_dag.print_hello 2024-01-01T00:00:00+00:00 [queued]>
[2025-04-01 21:30:17,211] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2025-04-01 21:30:17,212] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2025-04-01 21:30:17,212] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2025-04-01 21:30:17,222] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): print_hello> on 2024-01-01T00:00:00+00:00
[2025-04-01 21:30:17,228] {standard_task_runner.py:52} INFO - Started process 55 to run task
[2025-04-01 21:30:17,233] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_test_dag', 'print_hello', '2024-01-01T00:00:00+00:00', '--job-id', '3', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tentando.py', '--cfg-path', '/tmp/tmpj_6nazm4', '--error-file', '/tmp/tmpgu53olb3']
[2025-04-01 21:30:17,234] {standard_task_runner.py:77} INFO - Job 3: Subtask print_hello
[2025-04-01 21:30:17,295] {logging_mixin.py:104} INFO - Running <TaskInstance: simple_test_dag.print_hello 2024-01-01T00:00:00+00:00 [running]> on host 3921e1cf859c
[2025-04-01 21:30:17,341] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_test_dag
AIRFLOW_CTX_TASK_ID=print_hello
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T00:00:00+00:00
[2025-04-01 21:30:17,342] {logging_mixin.py:104} INFO - Hello, Airflow!
[2025-04-01 21:30:17,343] {python.py:118} INFO - Done. Returned value was: None
[2025-04-01 21:30:17,355] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=simple_test_dag, task_id=print_hello, execution_date=20240101T000000, start_date=20250401T213017, end_date=20250401T213017
[2025-04-01 21:30:17,386] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01 21:30:17,408] {local_task_job.py:146} INFO - Task exited with return code 0
